cf-serverd/server_transform.c

```diff
@@ -231,6 +231,8 @@ void KeepPromises(EvalContext *ctx, const Policy *policy, GenericAgentConfig *co
     roles_acl     = calloc(1, sizeof(*roles_acl));
     SV.path_shortcuts = StringMapNew();
 
+// gdb breakpoint for tests/acceptance/16_cf-serverd/keep_promises_fail.cf
+
     if (paths_acl    == NULL || classes_acl == NULL || vars_acl    == NULL ||
```

Comment:
@vpodzime how about this? I like it pretty well. Need to implement the other two instances and test on travis/jenkins

Reply:
I think using a more noticeable prefix like `GDB:` would be better. Also it would be nice if the comment was aligned with the code. But it's not a big deal.